doc/source/wsrep-system-index.rst

```diff
@@ -375,6 +375,57 @@ Possible values for this variable are:
 .. |binlog_format| replace:: ``binlog_format``
 .. _binlog_format: https://dev.mysql.com/doc/refman/5.7/en/binary-log-setting.html
 
+.. variable:: wsrep_min_log_verbosity
+
+   :cli: ``--wsrep-min-log-verbosity``
+   :conf: Yes
+   :scope: Global
+   :dyn: Yes
+   :default: 3
+
+This variable defines the *minimum* logging verbosity for the given
```

Comment:
Not exactly. This variable affects only logs that come from wsrep/galera replication part. Server logs are untouched.
Please see the description in the last comment of https://jira.percona.com/browse/PXC-2948
'wsrep_min_log_verbosity' defines the minimum logging verbosity from wsrep/galera. Its default value is 3